modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -1129,15 +1188,21 @@ public abstract class Base${schemaName}ResourceImpl
 		(${type})parameters.get("${value}")
 	<#else>
 		<#if type?contains("java.lang.Boolean")>
-			Boolean.parseBoolean(
+			<#assign privateParsersNeeded = privateParsersNeeded + ["boolean"] />
```

Comment:
I am not sure if Brian will like the naming `privateParsersNeeded`. Maybe something more appropriate would be `parserMethodDataTypes` ?

Reply:
Noted. Im going to change it right now.